pages/contribute.tsx

```diff
@@ -19,12 +33,140 @@ function Contribute(): JSX.Element {
+
+        <h2>contribution workflow</h2>
+        <p>
+          Repositories normally tell you how the steps you need to take to get to work, usually within their
+          README.mdor their CONTRIBUTING.md but generally, there&apos;s generally a standard couple steps
+          you&apos;ll have to take. For a detailed contribution guide, check out github&apos;s {' '}
+          <ELink link='https://docs.github.com/en/get-started/quickstart/contributing-to-projects'>
+            contribution guide
+          </ELink>!
+        </p>
+        <h3>forking and cloning a repo</h3>
+        <p>By clicking fork on a project that you want to contribute to, GitHub generates a personal copy of that
+          repo under your account. To clone it onto your computer, you can click the code button above the list of files
+          and run
+        </p>
+        <p><code>$ git clone git@github.com:uclaacm/opensource.git</code></p>
+        <p>to get it onto your machine.</p>
+        <h3>branching, making changes</h3>
+        <p>
+          Generally, when you want to make a specific change to a project, you make a branch,
+          so that you can work on a new feature without affecting other people&apos;s work.
+          To do so, run the <code>git checkout -b BRANCH_NAME</code> command
+          to generate a new branch or <code>git checkout BRANCH_NAME</code> to switch between branches.
+        </p>
+        <p>
+          After you&apos;ve made your changes, run {' '}
+          <code>git add .</code> then <code>git commit -m &quot;a short commit description&quot;</code> {' '}
+          to take a snapshot of all the changes you&apos;ve made and <code>git push</code> to push the changes you
+          made to GitHub.
+        </p>
+        <h3>making a pull request</h3>
+        <p>
+          Now that you&apos;ve successfully made the changes you want on your fork of the project,
+          if you head over to the original project repository and click <strong>Open a pull request</strong>,
+          you can put in a title and description of your changes.
+        </p>
+        <p>
+          After making a pull request, the maintainers of the project will check if your code is up to snuff
+          and request changes as necessary. Once they approve your changes however, you can merge your changes
+          in and you&apos;ve successfully contributed to the project!
+        </p>
+        <p className='action'>what are you waiting for? go work on some projects!</p>
+
+        <h2 id='good-first-issues'>
+          acm&apos;s projects w/ good first issues
+        </h2>
+        <p>
+          <span>acm currently has </span>
+          <strong>{displayedgfiProjects.length}</strong>
+          {
+            `
+          ${displayedgfiProjects.length>1 ? 'projects' : 'project' }
```

Comment:
```suggestion
          ${displayedgfiProjects.length !== 1 ? 'projects' : 'project' }
```

(what if there are zero projects)

Also, I think this is a good time to perhaps make a `pluralize` function, like [this SO post](https://stackoverflow.com/a/39835908)